### PR TITLE
Par 124

### PR DESCRIPTION
**Pull request by stephenb01**

### CODE REVIEW

File: web/modules/custom/par_flow_transition_business/src/Controller/ParFlowTransitionTaskListController.php [hunk 1]
```diff
@@ -21,20 +21,21 @@ class ParFlowTransitionTaskListController extends ParBaseController {
...
+    $primary_person = $primary_person_view_builder->view($par_data_primary_person, 'summary');
 
-    $build['premises'] = $premises_view_builder->view($par_data_premises, 'summary');
 
-    // Primary contact summary.
-    $par_data_primary_person = current($par_data_partnership->get('person')->referencedEntities());
-    $primary_person_view_builder = $par_data_primary_person->getViewBuilder();
+    $build['intro'] = [
+      '#markup' => t('Review and confirm the details of your partnership with %primary_authority by 14 September 2017',
+        ['%primary_authority' => $par_data_authority->authority_name->getValue()[0]['value']]),
```
Comment (kalpaitch) [inline web/modules/custom/par_flow_transition_business/src/Controller/ParFlowTransitionTaskListController.php]: Could we use `$par_data_authority->get('authority_name')->getString()` to get the name. Thanks.

File: web/modules/custom/par_flow_transition_business/src/Controller/ParFlowTransitionTaskListController.php [hunk 2]
```diff
@@ -21,20 +21,21 @@ class ParFlowTransitionTaskListController extends ParBaseController {
...
+    ];
 
-    $build['primary_contact'] = $primary_person_view_builder->view($par_data_primary_person, 'summary');
+    $build['contact'] = [
+      '#markup' => t("<h3>Main contact at the Authority</h3>") . render(($primary_person)),
```
Comment (kalpaitch) [inline web/modules/custom/par_flow_transition_business/src/Controller/ParFlowTransitionTaskListController.php]: Also @stemount found a slightly more efficient way to render the entity straight to markup. He will tell you.

We could also separate the title from the markup by doing `'#prefix' => "<h3>Main contact at the Authority</h3>"`

Comment (stephenb01) [inline web/modules/custom/par_flow_transition_business/src/Controller/ParFlowTransitionTaskListController.php]: That only appeared to work on a form not a controller. Tried it, but happy to apply the same changes.

File: web/modules/custom/par_flow_transition_business/src/Controller/ParFlowTransitionTaskListController.php (hunk 1 above)
Comment (stephenb01) [inline web/modules/custom/par_flow_transition_business/src/Controller/ParFlowTransitionTaskListController.php]: Ok

File: web/modules/custom/par_flow_transition_business/src/Controller/ParFlowTransitionTaskListController.php (hunk 2 above)
Comment (stemount) [inline web/modules/custom/par_flow_transition_business/src/Controller/ParFlowTransitionTaskListController.php]: yes, this is actually only available in ParBaseForm `$this->renderMarkupField(). sorting now.

File: web/modules/custom/par_flow_transition_business/src/Controller/ParFlowTransitionTaskListController.php [hunk 3]
```diff
@@ -21,20 +21,28 @@ class ParFlowTransitionTaskListController extends ParBaseController {
...
 
-    $build['primary_contact'] = $primary_person_view_builder->view($par_data_primary_person, 'summary');
+    $build['contact'] = [
+      '#type' => 'fieldset',
+      '#title' => t('Main contact at the Authority'),
+      '#collapsible' => FALSE,
+      '#collapsed' => FALSE,
+    ];
+
+    $build['contact']['name'] = [
```
Comment (kalpaitch) [inline web/modules/custom/par_flow_transition_business/src/Controller/ParFlowTransitionTaskListController.php]: Sorry, one last thing. I think this needs to be `$build['contact']['name'] = $primary_person;` Without the wrapping array??

Comment (stephenb01) [inline web/modules/custom/par_flow_transition_business/src/Controller/ParFlowTransitionTaskListController.php]: Waiting for Steve's fix before we complete!

**Pull request merged**